src/popup/popup.html

```diff
@@ -82,38 +82,46 @@ <h5 class="b-header margin-bottom-normal">Use</h5>
           <input type="checkbox" id="CC0" />
-          <label for="CC0"> <i class="icon cc-zero has-background-white margin-right-smaller"></i> CC0</label><br />
+          <label for="CC0"><i class="icon cc-logo has-background-white margin-right-smaller"></i> <i class="icon cc-zero has-background-white margin-right-smaller"></i> CC0<div class="tooltip" style="margin-left: 150px ;"><i class="icon question-circle"  ></i><span class="tooltiptext tooltip-right"><b> CC0</b><p><i class='icon cc-zero'></i> This work has been marked as dedicated to the public domain.</p><br/><p class="caption is-pulled-right margin-small">Read more about the tool <a href='#'>here</a></p></span>
+</div></label><br />
           <input type="checkbox" id="PDM" />
-          <label for="PDM"><i class="icon cc-pd has-background-white margin-right-smaller"></i>Public Domain Mark</label
+          <label for="PDM"> <i class="icon cc-logo has-background-white margin-right-smaller"></i><i class="icon cc-pd has-background-white margin-right-smaller"></i>Public Domain Mark<div class="tooltip" style="margin-left: 48px ;"><i class="icon question-circle"  ></i><span class="tooltiptext tooltip-right"><b> PDM</b><p><i class='icon cc-pd'></i> This work is marked as being in the public domain.</p><br/><p class="caption is-pulled-right margin-small">Read more about the tool <a href='#'>here</a></p></span>
+</div></label
           ><br />
           <input type="checkbox" id="BY" />
-          <label for="BY"><i class="icon cc-by has-background-white margin-right-smaller"></i>BY</label><br />
+          <label for="BY"> <i class="icon cc-logo has-background-white margin-right-smaller"></i><i class="icon cc-by has-background-white margin-right-smaller"></i>BY<div class="tooltip" style="margin-left: 165px ;"><i class="icon question-circle"  ></i><span class="tooltiptext tooltip-right"><b> License CC BY</b><p><i class='icon cc-by'></i> Credit the creator.</p><br/><p class="caption is-pulled-right margin-small">Read more about the tool <a href='#'>here</a></p></span>
+</div></label><br />
           <input type="checkbox" id="BY-SA" />
           <label for="BY-SA"
-            ><i class="icon cc-by has-background-white margin-right-smaller"></i
-            ><i class="icon cc-sa has-background-white margin-right-smaller"></i>BY-SA</label
+            ><i class="icon cc-logo has-background-white margin-right-smaller"></i><i class="icon cc-by has-background-white margin-right-smaller"></i
+            ><i class="icon cc-sa has-background-white margin-right-smaller"></i>BY-SA<div class="tooltip" style="margin-left: 125px ;"><i class="icon question-circle"  ></i><span class="tooltiptext tooltip-right"><b> License CC BY-SA</b><p><i class='icon cc-by'></i> Credit the creator.<br/><i class='icon cc-sa'></i> Share adaptations under the same terms.</p><br/><p class="caption is-pulled-right margin-small">Read more about the tool <a href='#'>here</a></p></span>
+</div></label
           ><br />
```

Comment:
We can do better than this. The license information, as well as the associated icon class names, are already present in [helper.js](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/helper.js#L89). They are used to dynamically fill out the license information in the image-detail section when the user clicks on the image.
![image-info](https://user-images.githubusercontent.com/34679965/98051172-4f857100-1e59-11eb-80d5-3437b9175e76.gif)
You can see the code in [imageDetailModule.js](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/imageDetailModule.js#L157) for your reference.

Reply:
Okay sir working on it .